src/main/java/com/trolley/trolley/Client.java

```diff
@@ -130,7 +130,7 @@ public String post(final String endPoint) throws Exception {
     public String patch(final String endPoint, final String body) throws Exception {
         String StringResponse = "";
         try {
-            final HttpClient httpclient = (HttpClient)HttpClients.createDefault();
+            final HttpClient httpclient = (HttpClient)HttpClients.createSystem();
```

Comment:
This is the same HTTP client just with enabled.

See https://cwiki.apache.org/confluence/display/HTTPCOMPONENTS/HttpClientConfiguration
and https://github.com/apache/httpcomponents-client/blob/22a257c2c227c63c20e59d8b561ffb187781223e/httpclient5/src/main/java/org/apache/hc/client5/http/impl/classic/HttpClients.java#L65